pages/news.py

```diff
@@ -16,6 +16,26 @@ def markdown_link(self):
 def news_items():
     donate_button = """<a class="btn button-custom btn-custom-two donate-button" href="donate.html">""" + _("Donate today.") + """</a>"""
     return [
+        NewsItem("20170301-1476", _("2017-03-01 - Freenet build 1476 released"),
+_("""
+Freenet 0.7.5 build 1476 is now available.
+
+Highlights:
+
+- efficiency improvements: routing for fast nodes and spare bitmaps
+- a new gif filter with improved security
+- improved maintainability by replacing custom code with standard code
+- show update info alongside bookmarks
+- ssl fixes
+- update plugins: Sharesite 0.4.4, Library v37, Freereader 6
```

Comment:
Perhaps those are among the most user-facing stuff and the changes should thus be clarified?

Anyway, please don't feel like I want to nitpick, just want to help with review - thanks for dealing with the website :)